src/main/java/spireMapOverhaul/zones/beastslair/BeastsLairZone.java

```diff
@@ -0,0 +1,142 @@
+    public static final String SaveID = makeID("BeastsLairBosses");
+    public static ArrayList<String> bossList = new ArrayList<>();
+
+    public BeastsLairZone() {
+        super(ID, Icons.MONSTER);
+        this.width = 1;
+        this.maxWidth = 1;
+        this.height = 1;
+        this.maxHeight = 1;
+        iconTexture = ImageMaster.MAP_NODE_ELITE;
+
+    }
+
+    public static void clearBossList() {
+        bossList.clear();
+    }
+
+
+    public static void initializeSaveFields() {
+        BaseMod.addSaveField(SaveID, new CustomSavable<ArrayList<String>>() {
+            @Override
+            public ArrayList<String> onSave() {
+                return BeastsLairZone.bossList;
+            }
+
+            @Override
+            public void onLoad(ArrayList<String> strings) {
+                if (strings != null) {
+                    BeastsLairZone.bossList = strings;
+                }
+            }
+        });
+    }
+
+    @Override
+    protected boolean canIncludeTreasureRow() {
+        return false;
+    }
+
+    @Override
+    protected boolean canIncludeFinalCampfireRow() {
+        return false;
+    }
+
+    @Override
+    protected boolean canIncludeEarlyRows() {
+        return false;
+    }
+
+    @Override
+    public void manualRoomPlacement(Random rng) {
+        for (MapRoomNode node : nodes) {
+            node.setRoom(new MonsterRoomElite());
+        }
+    }
+
+    @Override
+    public MonsterGroup changeEncounter(MonsterGroup monsterGroup) {
+        if (bossList.isEmpty()) {
+            return monsterGroup;
+        }
+
+        return MonsterHelper.getEncounter(bossList.get(AbstractDungeon.monsterRng.random(bossList.size() - 1)));
+    }
+
+    @Override
+    public int changeRareCardRewardChance(int rareCardRewardChance) {
+        return 50;
+    }
+
+    @Override
+    public int changeUncommonCardRewardChance(int uncommonCardRewardChance) {
+        return 50;
+    }
+
+    @Override
+    public ArrayList<RewardItem> getAdditionalRewards() {
+        ArrayList<RewardItem> rewards = new ArrayList<>();
+        rewards.add(new RewardItem(AbstractDungeon.returnRandomRelic(AbstractDungeon.returnRandomRelicTier())));
+
+        return rewards;
+    }
+
+    @Override
+    public List<ZoneEncounter> getEliteEncounters() {
+        ArrayList<ZoneEncounter> encounters = new ArrayList<>();
+        encounters.add(new ZoneEncounter("Act1FakeElite", 1, (BaseMod.GetMonster) () -> null));
+        encounters.add(new ZoneEncounter("Act2FakeElite", 2, (BaseMod.GetMonster) () -> null));
+        encounters.add(new ZoneEncounter("Act3FakeElite", 3, (BaseMod.GetMonster) () -> null));
+        return encounters;
+    }
+
+    @Override
+    public AbstractZone copy() {
+        return new BeastsLairZone();
+    }
+
+    @Override
+    public Color getColor() {
+        return Color.RED.cpy();
+    }
+
+    @Override
+    public boolean canSpawn() {
+        return AbstractDungeon.actNum >= 2;
```

Comment:
In endless mode this will mean this zone starts showing up in act 1 with act 3 bosses in it. If you want that, that's fine (endless mode is always a bit weird). If you don't want that, you can check `ActUtil.getRealActNum()` instead

Reply:
I think that makes sense to me because it the previous act, but I think could go either way. ill leave it for now and change it if it feels weird.